templates/index.html

```diff
@@ -1,61 +1,60 @@
 <!DOCTYPE html>
 <html>
 	<head>
+		<title>{% block title %}{{ config.title }}{% endblock title %}</title>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta content="{{ config.description }}" name="description">
+		<meta content="software development, software engineering, podcast" name="keywords">
 
-	  <link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x16-favicon.png') }}" sizes="16x16" />
-	  <link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x32-favicon.png') }}" sizes="32x32" />
-	  <link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x64-favicon.png') }}" sizes="64x64" />
-	  <link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x256-favicon.png') }}" sizes="256x256" />
-	  <link rel="apple-touch-icon" type="image/png" href="{{ get_url(path='favicons/x256-favicon.png') }}" />
-
-	  {% block meta %}
-
-    <meta content="{{ config.description }}" name="description">
-    <meta content="software development, software engineering, podcast" name="keywords">
+		<link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x16-favicon.png') | safe }}" sizes="16x16" />
+		<link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x32-favicon.png') | safe }}" sizes="32x32" />
+		<link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x64-favicon.png') | safe }}" sizes="64x64" />
+		<link rel="shortcut icon" type="image/png" href="{{ get_url(path='favicons/x256-favicon.png') | safe }}" sizes="256x256" />
+		<link rel="apple-touch-icon" type="image/png" href="{{ get_url(path='favicons/x256-favicon.png') | safe }}" />
+		<link rel="alternate" type="application/rss+xml" title="Software Sessions" href="https://feeds.transistor.fm/software-sessions" />
+		<link rel="stylesheet" href="{{ get_url(path='style.css') | safe }}">
 
-  	<!-- Facebook -->
-  	<meta property="og:url" content="{{ config.base_url }}">
-  	<meta property="og:type" content="website">
-  	<meta property="og:site_name" content="{{ config.title }}">
-  	<meta property="og:description" content="{{ config.description }}">
-  	<meta property="og:title" content="{{ config.title }}">
-  	<meta property="og:image" content="{{ get_url(path='images/top-banner.png') }}">
-  	<!-- /Facebook -->
+		{% set social_title = config.title %}
+		{% if page %}
+			{% set social_title =  page.title ~ " | " ~ config.title %}
+		{% elif section and section.title %}
+			{% set social_title =  section.title ~ " | " ~ config.title %}
+		{% endif %}
```

Comment:
This way we don't have to repeat the meta on pages and section, it's only defined in the index.

Reply:
Ah yes, this is much cleaner. Hadn't thought to check if it was on a page or section.